packages/berry-fslib/package.json

```diff
@@ -15,7 +15,8 @@
     "postpack": "rm -rf lib"
   },
   "publishConfig": {
-    "main": "./lib/index.js"
+    "main": "./lib/index.js",
+    "typings": "./lib/index.d.ts"
```

Comment:
It made sense for me to add typings to this package, feel free to disagree :) 

I wanted to add typings to `@berry/libzip` as well but it doesn't work out of the box due to the `.d.ts` file in the `sources` folder.

Reply:
Yes but no&hellip; The documentation (quoted below) says they do that for top-level `index.js` and top-level `index.d.ts` but they also say please mark it in the package manifest.

Also note they support `typings` and `types`, so best I'd best add the `types` to the hook as well

---

If your package has a main .js file, you will need to indicate the main declaration file in your package.json file as well. Set the types property to point to your bundled declaration file. For example:

{
    "name": "awesome",
    "author": "Vandelay Industries",
    "version": "1.0.0",
    "main": "./lib/main.js",
    "types": "./lib/main.d.ts"
}
Note that the "typings" field is synonymous with "types", and could be used as well.

Also note that if your main declaration file is named index.d.ts and lives at the root of the package (next to index.js) you do not need to mark the "types" property, though it is advisable to do so.

~ https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html